src/SDKs/CognitiveServices/dataPlane/Language/SpellCheck/BingSpellCheck/Generate.ps1

```diff
@@ -0,0 +1 @@
+powershell.exe -ExecutionPolicy Bypass -NoLogo -NonInteractive -NoProfile -File "..\..\..\..\..\..\..\tools\generateTool.ps1" -ResourceProvider  "cognitiveservices\data-plane\SpellCheck" -PowershellInvoker  -AutoRestVersion "latest"
```

Comment:
Replace this with 
`powershell.exe -ExecutionPolicy Bypass -NoLogo -NonInteractive -NoProfile -File "..\..\..\..\..\..\..\tools\generateTool.ps1" -ResourceProvider  "cognitiveservices/data-plane/SpellCheck/BingSpellCheck" -PowershellInvoker  -AutoRestVersion "latest"`